.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+ - '2.7'
+ - '3.5'
+sudo: false
+branches:
+  only:
+  - master
+install:
+- pip install -r requirements/tox.txt
+- pip install tox-travis
```

Comment:
There's an [example in django-user-tasks](https://github.com/edx/django-user-tasks/blob/master/.travis.yml), if that helps.